index.js

```diff
@@ -12,12 +12,31 @@ module.exports = function(content) {
   var config = loaderUtils.getLoaderConfig(this, "imageWebpackLoader");
   var options = {
     bypassOnDebug: config.bypassOnDebug || false,
+    interlaced: config.interlaced || false,
+    optimizationLevel: config.optimizationLevel || 3,
     gifsicle: config.gifsicle || {},
     mozjpeg: config.mozjpeg || {},
     pngquant: config.pngquant || {},
     optipng: config.optipng || {},
     svgo: config.svgo || {}
   };
+  // Remove line 15-16 and 23-39 in new major version
+  if(config.gifsicle === undefined){
```

Comment:
Users that just use defaults will also get the warning this way. Thought it might be useful to let them know too.

Should we warn users that explicitly set interlaced & optimizationLevel but not those who use the defaults?